### ISSUE TITLE
I would like to know the hyperparameter setting of 45.2mm  model

### ISSUE THREAD

**Issue by ZivDk** [QitaoZhao/PoseFormerV2#37]
recently , I tried to train the model , frame = 243 central frame = 27 coefficient = 27 .  I expect to get the result which is close to 45.2mm , because accuracy of my model is only about 46.6mm

I really want to know how to train the similar model like yours. please tell me how can I set correctly to get this outcome

thank you very much for your help

**Comment by QitaoZhao** [comment]
Hi, can you try `bs: 256, lr: 0.00016` (which I found from my old recordings)? We also found that the long-sequence model's performance is sensitive to hyperparameters. You may need to tune the learning rate; a small batch size empirically produces better results.

**Comment by ZivDk** [comment]
thank you it helps me a lot
